lib/git_reflow.rb

```diff
@@ -45,17 +47,56 @@ def review(options = {})
 
       existing_pull_request = git_server.find_open_pull_request( from: current_branch, to: options['base'] )
       if existing_pull_request
-        puts "A pull request already exists for these branches:"
+        say "A pull request already exists for these branches:", :notice
         existing_pull_request.display_pull_request_summary
         ask_to_open_in_browser(existing_pull_request.html_url)
       else
-        pull_request = git_server.create_pull_request(title: options['title'],
-                                                      body:  options['body'],
-                                                      head:  "#{remote_user}:#{current_branch}",
-                                                      base:  options['base'])
+        unless options['title'] || options['message']
+          pull_request_msg_file = "#{GitReflow.git_root_dir}/.git/GIT_REFLOW_PR_MSG"
+          default_editor        = "#{ENV['EDITOR']}"
+
+          if default_editor.empty?
+            default_editor = 'vi'
+          end
+
+          File.open(pull_request_msg_file, 'w') do |file|
+            file.write(GitReflow.current_branch)
+          end
 
-        puts "Successfully created pull request ##{pull_request.number}: #{pull_request.title}\nPull Request URL: #{pull_request.html_url}\n"
-        ask_to_open_in_browser(pull_request.html_url)
+          GitReflow.run("#{default_editor} #{pull_request_msg_file}", with_system: true)
+
+          pr_msg = File.read(pull_request_msg_file).split(/[\r\n]|\r\n/).map(&:strip)
+          title  = pr_msg.shift
+
+          File.delete(pull_request_msg_file)
+
+          unless pr_msg.empty? 
+            pr_msg.shift if pr_msg.first.empty?
+          end
+
+          options['title'] = title
+          options['body']  = "#{pr_msg.join("\n")}\n"
+
+          puts "\nReview your PR:\n"
+          puts "--------\n"
+          puts "Title:\n#{options['title']}\n\n"
+          puts "Body:\n#{options['body']}\n"
+          puts "--------\n"
+
+          create_pull_request = ask("Submit pull request? (Y)") =~ /y/i
+        end
+
+        if create_pull_request
+          pull_request = git_server.create_pull_request(title: options['title'],
```

Comment:
if `options['message']` is defined, then `options['title']` will always be blank